module/VuFind/src/VuFind/ILS/Driver/Factory.php

```diff
@@ -57,6 +57,21 @@ public static function getAleph(ServiceManager $sm)
     }
 
     /**
+     * Factory for Alma driver.
+     *
+     * @param ServiceManager $sm Service manager.
+     *
+     * @return Alma
+     */
+    public static function getAlma(ServiceManager $sm)
+    {
+        return new Alma(
+            $sm->getServiceLocator()->get('VuFind\DateConverter'),
```

Comment:
If I'm reading everything correctly, the Alma driver currently has no dependencies, so you can remove these constructor parameters for now. You might eventually need the DateConverter for formatting due dates (among other things), and maybe the CacheManager could somehow be used to help performance, but we might as well take them away until we actually need them.